Formula/odin.rb

```diff
@@ -20,19 +14,23 @@ class Odin < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux: "1a2aed348bd038115e6581e293a69291eca7a2ef89a59589faf36d9ed81089d1"
   end
 
-  # Check if this can be switched to `llvm` at next release
-  depends_on "llvm@11"
+  on_macos do
+    depends_on "llvm@13"
```

Comment:
Using `llvm` will probably break with `llvm@14` #97618.
So, PR will be blocked until `llvm` PR is merged and then `llvm@13` is added.